grid/address/geocoding/maps/locationiq.suma

```diff
@@ -0,0 +1,168 @@
+profile = "address/geocoding@3.1"
+provider = "locationiq"
+
+// https://locationiq.com/docs
+
+map Geocode {
+  address = input.query || `${input.streetAddress || ""} ${input.addressLocality || ""} ${input.addressRegion || ""} ${input.postalCode || ""} ${input.addressCountry || ""}`
+
+  http GET "/v1/search" {
+    security "apikey"
+    
+    request {
+      query {
+        format = 'json'
+        q = input.query
+        street = input.streetAddress
+        city = input.addressLocality
+        state = input.addressRegion
+        country = input.addressCountry
+        postalcode = input.postalCode
+      }
+    }
+
+    response 200 "application/json" {
+      return map error if (statusCode !== 200 || !body.length) {
```

Comment:
How can `statusCode !== 200` in a `response 200` block?

Reply:
Weird, right? I just copied this from Nominatim map.